apps/meteor/app/federation-v2/server/infrastructure/rocket-chat/hooks/index.ts

```diff
@@ -47,6 +51,10 @@ export class FederationHooks {
 		callbacks.add(
 			'federation.beforeAddUserAToRoom',
 			(params: { user: IUser | string; inviter: IUser }, room: IRoom): void => {
+				if (!params || !params.user || !room || !params.inviter) {
```

Comment:
I believe we need to remove the `!params.inviter` from here.

Reply:
Fixed!